crates/trie/trie/src/hashed_cursor/post_state.rs

```diff
@@ -168,170 +211,52 @@ where
     /// If the cursor is positioned at the entry, return the entry with next greater key.
     /// Returns [None] if the previous memoized or the next greater entries are missing.
     ///
-    /// NOTE: This function will not return any entry unless [`HashedCursor::seek`] has been
-    /// called.
+    /// NOTE: This function will not return any entry unless [`HashedCursor::seek`] has been called.
```

Comment:
We should use the debug_assert on `self.seeked`, and this comment can be updated to indicate that it will panic